tsdb/chunks/head_chunks.go

```diff
@@ -273,6 +273,26 @@ func NewChunkDiskMapper(reg prometheus.Registerer, dir string, pool chunkenc.Poo
 	return m, m.openMMapFiles()
 }
 
+// Chunk encodings for out-of-order chunks.
+// These encodings must be only used by the Head block for its internal bookkeeping.
+const (
+	OutOfOrderMask        = 0b10000000
+	RestoreOutOfOrderMask = 0b01111111
+	// EncOOOXOR      = chunkenc.EncXOR | OutOfOrderMask
```

Comment:
Can we remove this unused const or is it signifying something?

Reply:
sure, I would do.